src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <container class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </container>
+      <container class="stars stars--1">
+        <div class="stars__star-active"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </container>
+      <container class="stars stars--2">
+        <div class="stars__star-active"></div>
+        <div class="stars__star-active"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </container>
+      <container class="stars stars--3">
+        <div class="stars__star-active"></div>
+        <div class="stars__star-active"></div>
+        <div class="stars__star-active"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </container>
+      <container class="stars stars--4">
+        <div class="stars__star-active"></div>
+        <div class="stars__star-active"></div>
+        <div class="stars__star-active"></div>
+        <div class="stars__star-active"></div>
+        <div class="stars__star"></div>
+      </container>
+      <container class="stars stars--5">
+        <div class="stars__star-active"></div>
+        <div class="stars__star-active"></div>
+        <div class="stars__star-active"></div>
+        <div class="stars__star-active"></div>
+        <div class="stars__star-active"></div>
```

Comment:
The same here

Reply:
But how then can I add two different pictures to the same class?

Reply:
> But how then can I add two different pictures to the same class?
in class="stars__star-active" all stars are yellow, in another class they are gray